integration/spark/shared/src/main/java/io/openlineage/spark/agent/lifecycle/plan/column/TransformationInfo.java

```diff
@@ -40,15 +40,27 @@ public static TransformationInfo identity() {
   }
 
   public static TransformationInfo transformation() {
-    return new TransformationInfo(Types.DIRECT, Subtypes.TRANSFORMATION, "", false);
+    return transformation(false);
+  }
+
+  public static TransformationInfo transformation(Boolean isMasking) {
+    return new TransformationInfo(Types.DIRECT, Subtypes.TRANSFORMATION, "", isMasking);
   }
 
   public static TransformationInfo aggregation() {
-    return new TransformationInfo(Types.DIRECT, Subtypes.AGGREGATION, "", false);
```

Comment:
Could you provide some docs for those method explaining/giving an example on how an aggregation transformation type can be masking or not?

Reply:
well the only one masking, aggregation transformation I could think of is `count` as it completely disregards the value in the column no matter what

Reply:
added some javadoc